### PR TITLE
Addition of new CHWS types (CHWS_RWTC_CHWVM and CHWS_SWTM_WDPC_BYPVPM).

### PR DESCRIPTION
**Pull request by ghairapetian**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/CHWS.yaml [hunk 1]
```diff
@@ -214,6 +214,25 @@ CHWS_SWTC_WDPC2X_BYPVPM_WFRM_CLPM:
   - WFRM
   - CLPM
 
+CHWS_RWTC_CHWVM:
+  id: "16842546300862857217"
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/CHWS.yaml:218]: Please remove ID. New entity types do not require IDs (our system sets the IDs automatically).

File: ontology/yaml/resources/HVAC/entity_types/CHWS.yaml [hunk 2]
```diff
@@ -214,6 +214,25 @@ CHWS_SWTC_WDPC2X_BYPVPM_WFRM_CLPM:
...
+  description: "Chilled water system with return temperature control and chilled water valve monitoring."
+  is_canonical: true
+  implements:
+  - CHWS
+  - RWTC
+  - CHWVM
+
+CHWS_SWTM_WDPC_BYPVPM:
+  id: "16842546300862857218"
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/CHWS.yaml:227]: Please remove ID.

**Review by tasodorff — APPROVED** [review]
Remove IDs. Everything else looks good.

**Review by tasodorff — CHANGES_REQUESTED** [review]
Meant to request changes instead of approve; please remove IDs.

**Comment by ghairapetian** [comment]
Something messy happened with the rebase. I will close this pull request and reopen a new one. Sorry for that.

**Pull request closed without merging**